applications/luci-app-dcwapd/Makefile

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (C) 2019 EWSI
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luci-app-dcwapd
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/luci-app-dcwapd
+  SECTION:=luci
+  CATEGORY:=LuCI
+  SUBMENU:=3. Applications
+  DEPENDS:=+luci +dcwapd
+  TITLE:=Dual Channel Wi-Fi AP Daemon configuration module
+endef
+
+define Package/luci-app-dcwapd/description
+Luci web-admin support for the Dual Channel Wi-Fi AP Daemon
+endef
+
+define Build/Compile
+endef
+
+define Package/luci-app-dcwapd/install
+  $(CP) ./files/* $(1)
+endef
+
+$(eval $(call BuildPackage,luci-app-dcwapd))
```

Comment:
Please have a look on other Makefile in this application folder. Use this as a template.